deeplake/core/dataset/dataset.py

```diff
@@ -1832,7 +1835,7 @@ def has_head_changes(self):
             self.version_state, self.storage
         )
 
-    def _acquire_lock(self, timeout: Optional[int] = None):
+    def _acquire_lock(self, timeout: Optional[int] = 0):
```

Comment:
why this change? it seems like there's a comparison with `None` below.

Reply:
Removed the method as it was no longer used